### PR TITLE
[flink]validateMySqlConfig

### PR DESCRIPTION
**Pull request by ymZhao1001**
[flink]

### Purpose

validateMySqlConfig

### CODE REVIEW

File: paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java [hunk 1]
```diff
@@ -202,4 +205,37 @@ static MySqlSource<String> buildMySqlSource(Configuration mySqlConfig) {
                 new JsonDebeziumDeserializationSchema(true, customConverterConfigs);
         return sourceBuilder.deserializer(schema).includeSchemaChanges(true).build();
     }
+
+    private static void validateMySqlConfig(Configuration mySqlConfig) {
```
Comment (JingsongLi) [inline paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java:209]: Can we invoke `org.apache.flink.table.factories.FactoryUtil.validateFactoryOptions`?

File: paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java [hunk 2]
```diff
@@ -202,4 +205,37 @@ static MySqlSource<String> buildMySqlSource(Configuration mySqlConfig) {
...
+        checkArgument(
+                mySqlConfig.get(MySqlSourceOptions.HOSTNAME) != null
```
Comment (JingsongLi) [inline paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java:211]: `mySqlConfig.contains(MySqlSourceOptions.HOSTNAME)`
Just check `contains` is OK, if it is a empty string, detailed message will be reported in mysql client.

Comment (ymZhao1001) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java]: remove the check whether it is empty

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**